x/wasmplus/keeper/querier_test.go

```diff
@@ -21,19 +24,80 @@ func TestQueryInactiveContracts(t *testing.T) {
 	example2 := InstantiateHackatomExampleContract(t, ctx, keepers)
 	ctx = ctx.WithBlockHeight(ctx.BlockHeight() + 1)
 
+	// Address order of contracts is ascending order of byte array whose address is decoded by bech32
+	expAddrs := CreateOrderedAddresses(example1.Contract.Bytes(), example2.Contract.Bytes())
```

Comment:
Is it necessary to add more contracts in this test case?

Reply:
Thanks. Three contracts seem to be able to test the order. It would be nice if it was added.